ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,31 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        _node_status = _res.get(nodeid)
+        if _node_status.lower() == NODE_STATUSES.ONLINE.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is already in Online status"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY_WITH_RESOURCES_RUNNING.value.lower():
+            return {"status": "Succeeded", "msg": f"Node {nodeid}, is going in standby mode, "
+                                                  f"We need to wait to complete the resource shutdown"}
+        elif _node_status.lower() == NODE_STATUSES.STANDBY.value.lower():
+            # make node unstandby
+            if self.heal_resource(nodeid):
+                _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_UNSTANDBY.replace("<node>", nodeid),
+                                                           check_error=False)
+                return {"status": "InProgress", "msg": f"Node {nodeid} : Node was in standby mode, "
```

Comment:
can we have this in const like NODE_STATUSES

InProgress
Success
Failed

Reply:
Sure